### PR TITLE
optimize: Removed Reloadable from the redis/db SessionManager.

### PR DESCRIPTION
**Pull request by wangliang181230**
### Ⅰ. Describe what this PR did
从redis和db模式的SessionManager中移除Reloadable接口实现。因为它们并不需要reload。

从PR #2828 拆分出来的。

### CODE REVIEW

File: server/src/main/java/io/seata/server/session/SessionHolder.java [hunk 1]
```diff
@@ -117,39 +117,52 @@ public static void init(String mode) throws IOException {
...
+                            ROOT_SESSION_MANAGER.removeGlobalSession(globalSession);
+                            if (LOGGER.isInfoEnabled()) {
+                                LOGGER.info("Remove global session succeeded, xid = {}, status = {}", globalSession.getXid(), globalSession.getStatus());
+                            }
+                        } catch (Exception e) {
+                            LOGGER.error("Remove global session failed, xid = {}, status = {}", globalSession.getXid(), globalSession.getStatus(), e);
+                        }
+                        break;
+                    case AsyncCommitting:
+                        if (storeMode == StoreMode.FILE) {
```
Comment (l81893521) [inline server/src/main/java/io/seata/server/session/SessionHolder.java:147]: Why only file mode can reload this status of global session?
```
AsyncCommitting
CommitRetrying
TimeoutRollbackRetrying
Begin
```

Comment (l81893521) [reply server/src/main/java/io/seata/server/session/SessionHolder.java:147]: If say so, this hold method just use for file mode?

File: server/src/main/java/io/seata/server/session/SessionHolder.java [hunk 2]
```diff
@@ -117,83 +118,122 @@ public static void init(String mode) throws IOException {
...
+
     /**
      * Reload.
      */
-    protected static void reload() {
+    protected static void reload(StoreMode storeMode) {
         if (ROOT_SESSION_MANAGER instanceof Reloadable) {
-            ((Reloadable)ROOT_SESSION_MANAGER).reload();
+            ((Reloadable) ROOT_SESSION_MANAGER).reload();
+        }
```
Comment (slievrly) [inline server/src/main/java/io/seata/server/session/SessionHolder.java:132]: So only in file mode do we need to reload, right?

**Review by slievrly — APPROVED** [review]
LGTM

File: server/src/main/java/io/seata/server/session/SessionHolder.java (hunk 2 above)
Comment (wangliang181230) [reply server/src/main/java/io/seata/server/session/SessionHolder.java:132]: yes

Comment (wangliang181230) [reply server/src/main/java/io/seata/server/session/SessionHolder.java:132]: redis and db mode, only need to remove the global sessions in error state.

File: core/src/main/java/io/seata/core/constants/ConfigurationKeys.java [hunk 3]
```diff
@@ -25,7 +25,7 @@
     /**
      * The constant SEATA_PREFIX.
      */
-    public static final String SEATA_PREFIX = "seata.";
```
Comment (wangliang181230) [inline core/src/main/java/io/seata/core/constants/ConfigurationKeys.java:28]: 多余的关键字。

**Review by l81893521 — APPROVED** [review]
LGTM

**Pull request merged**